### PR TITLE
Switch to V2

### PR DESCRIPTION
**Pull request by christianhaller**
with shiny screenshots (keep in mind this bug https://groups.google.com/d/msg/google-api-python-client/DjbJ0BwjbPo/lCEUt-dmnRwJ )

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -117,6 +117,7 @@ $ psi --help
     --key          Google API Key. By default the free tier is used.
     --strategy     Strategy to use when analyzing the page: mobile|desktop
     --format       Output format: cli|json|tap
+    --screenshot   jpg screenshot
```
Comment (hemanth) [inline README.md]: :+1:

**Comment by addyosmani** [comment]
Will require some local testing, but changes initially LGTM. Thanks!

**Comment by christianhaller** [comment]
That's the fix screenshot
````
var screenshot = new Buffer(data.screenshot.data.replace(/_/g,'/');, 'base64');

**Comment by addyosmani** [comment]
Changes here LGTM. It's also great to see screenshot support land as a flag. @hemanth does V2 support impact https://github.com/addyosmani/psi/pull/59 at all?

**Comment by hemanth** [comment]
@addyosmani Nope, It doesn't impact #59, there might be few merge conflicts that's it.

**Pull request merged**